### DISCUSSION TITLE
Firebase Functions

### DISCUSSION THREAD

**Discussion by GutoFrr** [honojs/hono#1469] [Q&A]
Hello! There is a way to deploy my Hono API to Firebase Functions? Or any way to deploy a my API built on top of Node.js?

Thanks!

**Comment by alexfriesen** [comment]
I tried Firebase a while ago and it was very tricky. So I just deployed a node server with Cloud Run instead.
You can find the node Adapter here: https://github.com/honojs/node-server

**Reply by GutoFrr** [reply]
Great idea. I'm gonna try this and then I'll give feedback here. Thank you!

**Comment by RobertSasak** [comment]
It is possible to used as described here https://github.com/honojs/hono/issues/1358

**Comment by 0x80** [comment]
Considering that Firebase Functions is using Express under the hood, would we lose all performance benefits of Hono?

I am still interested in embedding it for HonoRPC, but if the overhead is substantial, I will consider using Cloud Run deployments instead.
